qemu/tests/cfg/balloon_small_size.cfg

```diff
@@ -0,0 +1,12 @@
+- balloon_small_size:
+    virt_test_type = qemu
+    no Win2000
+    only Windows
```

Comment:
The difference between linux and windows guest for this case is just the expect_mem .